### DISCUSSION TITLE
What is the best way for P2 to know who is the attacker or attacker ID in Ikemen?

### DISCUSSION THREAD

**Discussion by beterhans** [ikemen-engine/Ikemen-GO#1052] [Q&A] [answered]
current workaround for me
put attacker id in hitdef parameters

let victim to get it by using gethitvar()
but sometime cause issue in ikemen

**Comment by potsmugen** [comment, marked as answer]
Ikemen introduced a new GetHitVar(ID) trigger. It returns the ID of the player that hit you.
